policyengine_us/variables/gov/states/hi/tax/income/credits/hi_low_income_household_renters/hi_low_income_household_renters_tax_credit.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class hi_low_income_household_renters_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii low income household renters tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "hi_lihrtc_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.lihrtc
+
+        tax_before_credit = tax_unit("hi_income_tax_before_credits", period)
```

Comment:
Remove this from the formula. If this is a non-refundable credit we'll cap it along with other non-refundable credits in the `hi_tax_before_refundable_credits` formula.